README.md

```diff
@@ -248,16 +250,36 @@ The name format will include the environment you're after at the end of the name
 
 ```shell script
 az aks get-credentials \
-  -g pins-uks-k8s-prod \
-  -n pins-uks-k8s-1234-prod \
+  -g pins-uks-k8s-dev \
+  -n pins-uks-k8s-5883-dev \
   --overwrite-existing
 ```
 
 This will save the config file to `~/.kube/config`. Now you have this file, you will be able to connect to the
 Kubernetes cluster.
 
 ```shell script
-kubectl logs -f -n app-prod app-form-web-app-1111111111-aaaaa
+kubectl logs -f -n app-dev app-form-web-app-1111111111-aaaaa
+```
+
+The available namespaces can be listed with the command:
+```shell script
+kubectl get namespace
```

Comment:
Does this command work for you now? Or is that in on the assumption I get it working from my end?

Reply:
The command is correct. Whether it works will depend on individual permissions. It was not working for me when I checked last.